services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -263,7 +263,20 @@ async def create_file_upload_links(
             )
 
             # ensure file is deleted first in case it already exists
-            await self.delete_file(user_id=user_id, file_id=file_id)
+
+            #   _   _  ___ _____ _____
+            #  | \ | |/ _ \_   _| ____|
+            #  |  \| | | | || | |  _|
+            #  | |\  | |_| || | | |___
+            #  |_| \_|\___/ |_| |_____|
+            # NOTE: (a very big one)
+            # `enforce_access_rights` is set to False because if the service is shared
+            # with a collaborator, the collaborator does not have access to delete
+            # the file. This happened because we are using project access rights to enforce
+            # them on the data. This should change and we should be using data access rights
```

Comment:
is there an issue for that change? 
NOTE also that if you allow to bypass these check (i.e. `enforce_access_rights=True`) this might be used somewhe. Therefore, perhaps the big note should also be in `delete_file`?

Reply:
agree, please create an issue for creating data access rights and reference it her and in delete_file.
Nobody shall enforce these rights in any other use case.